ansible/roles_ocp_workloads/ocp4_workload_shared_cluster_access/tasks/post_workload.yml

```diff
@@ -1,9 +1,36 @@
 ---
 # Implement your Post Workload deployment tasks here
-# --------------------------------------------------
+#--- --------------------------------------------------
+- name: Get console route
+  k8s_info:
+    api_version: route.openshift.io/v1
+    kind: Route
+    namespace: openshift-console
+    name: console
+  register: routeconsole
 
+- name: API for CLI
+  command: oc whoami --show-server
+  register: showserver
 
+- name: Set webconsole
+  set_fact:
+    webconsole: "http://{{ routeconsole.resources[0].spec.host }}"
+  when: showserver.changed
```

Comment:
Why this? We want this fact set no matter what, no?

Reply:
My mistake, should be fixed now.